site/src/api/typesGenerated.ts

```diff
@@ -445,6 +446,7 @@ export interface DeploymentValues {
   readonly allow_workspace_renames?: boolean;
   readonly healthcheck?: HealthcheckConfig;
   readonly cli_upgrade_message?: string;
+  readonly terms_of_service_link?: string;
```

Comment:
Why is this here and also in the `AuthMethods` type?

Reply:
- it has to be here, because this is the type that all of our config from env variables and args gets loaded into
- we want to keep /users/authmethods as the only query that the login screen needs to make, so the custom terms of service gets served from there